lib/logstash/pipeline.rb

```diff
@@ -74,31 +83,22 @@ def filters?
   end
 
   def run
+    LogStash::Util.set_thread_name(">lsipeline")
```

Comment:
small typo `lsipeline`? I think we can use the full name here `logstash-pipeline`